core/server/services/members/api.js

```diff
@@ -37,6 +37,17 @@ function getMember(data, options) {
     });
 }
 
+function deleteMember(options) {
+    options = options || {};
+    return models.Member.destroy(options).catch(models.Member.NotFoundError, () => {
+        throw new common.errors.NotFoundError({
```

Comment:
This is a HTTP specific error, maybe it should be handled/thrown in the api dir? :fried_shrimp:

Reply:
🤔 Since the `NotFoundError` is attached to `models.Member` I think it makes more sense to throw it from where model is called directly, otherwise would need to add `models` to members api dir which didn't seem right.